### DISCUSSION TITLE
Lumped port in PyAEDT

### DISCUSSION THREAD

**Discussion by 88v88** [ansys/pyaedt#5148] [Q&A] [answered]
Hi, I am trying to assign a lumped port in HFSS using `lumped_port` PyAEDT. However, I get this error when I select a face using `get_faceid_from_position` and specify an integration line just like how I assign wave ports with `wave_port`. The face is valid and  I tried looking in the documentation and I could not generate a lumped port without errors. What is the difference in assigning geometries to lumped ports compared to wave ports? 
   ` [error] Script macro error: Port '2': A geometry selection is required for assignment. (12:58:17 PM  Sep 09, 2024)`
Thanks.

**Comment by Samuelopez-ansys** [comment]
Hi @88v88,

Please could you provide a snipped code to reproduce the issue?

Thanks

**Reply by 88v88** [reply, marked as answer]
Hi @Samuelopez-ansys , I resolved the problem by assigning the lumped port to a rectangle drawn on a face instead of selecting a face on an object, thanks.
